nekoyume/Assets/_Scripts/Game/Character/Player.cs

```diff
@@ -410,6 +405,11 @@ private void UpdateEar()
         /// <param name="customizeIndex">0~9</param>
         public void UpdateEarByCustomizeIndex(int customizeIndex)
         {
+            if (IsFullCostumeEquipped || SpineController is null)
```

Comment:
I think. it would be good if L.499 and L.549 are the same

Reply:
Thanks!

```suggestion
            if (IsFullCostumeEquipped || !SpineController)
```